.github/workflows/Release.yaml

```diff
@@ -134,4 +134,16 @@ jobs:
           branch: ${{ github.ref }}
           message: 'ci: automatically update oci/${{ inputs.oci-image-name }}/_releases.json, from ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}'
           files: oci/${{ inputs.oci-image-name }}/_releases.json
-          email: rocks-dev@lists.canonical.com
\ No newline at end of file
+          email: rocks-dev@lists.canonical.com
+      
+      - name: get the revision
+        id: get-revision
+        run: |
+          revision=$(cat "${{ steps.get-all-canonical-tags.outputs.canonical-tags-file }}" | tr ',' '\n' | sort -t '_' -k 2 -n | tail -n 1)
```

Comment:
this is not right. This pipeline can release multiple revisions, see https://github.com/canonical/oci-factory/blob/main/src/image/release.py#L212.

Here you are assuming it can only release 1. Plus, you're also relying on information that sits in Swift, and not the actual information that is being used to trigger this workflow - the image trigger

Reply:
:arrow_up: